### ISSUE TITLE
Update BatchNormalization documentation

### ISSUE THREAD

**Issue by vermouth1992** [tensorflow/tensorflow#43998]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue: 
https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization

### Clear description
The document should mention that the axis can take in a list of integers, not just an integer. I tested it and it is already implemented in the TensorFlow. I was not aware of it and used reshape and transpose, which is inefficient.

### Correct links

Is the link to the source code correct?

### Parameters defined

Are all parameters defined and formatted correctly?

The axis can be a list of integers.

### Returns defined

Are return values defined?

### Raises listed and defined

Are the errors defined? For example,
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/feature_column/categorical_column_with_vocabulary_file#raises

### Usage example

Is there a usage example?

See the API guide: https://www.tensorflow.org/community/contribute/docs_ref
on how to write testable usage examples.

### Request visuals, if applicable

Are there currently visuals? If not, will it clarify the content?

### Submit a pull request?

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs,
docs API guide: https://www.tensorflow.org/community/contribute/docs_ref and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

**Comment by Saduf2019** [comment]
@vermouth1992 
Could you please explain the issue faced.

**Comment by ymodak** [comment]
Thanks for your issue. The docs are now updated.
https://github.com/tensorflow/tensorflow/blob/9318f787e6d33adabc5c1b18c6663d1432f646f9/tensorflow/python/keras/layers/normalization.py#L83
